flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
this remain disappointing.

Reply:
Flutter doesn't support cross-compiling and I haven't set up a GitHub Action for an aarch64 Linux build yet, so for now it's x86_64-only.

Reply:
I checked just now, there actually aren't aarch64 Linux runners for GitHub Actions yet, but are planned for later this year. So until then there's nothing I can do if the FlatHub build process doesn't allow building from source with network access.